.github/workflows/README.md

```diff
@@ -0,0 +1,126 @@
+# AlphaX Release Process
+
+## Release a new version
+### Tagging of changes
+In order to have release notes automatically generated, changes need to be tagged with labels.
+The following labels are used (should be safe-explanatory):
+`breaking-change`, `bug`, `enhancement`.
+
+### Release a new version
+Note: Releases need to be done from the `main` branch.
+
+1. Bump the version locally to (e.g. to `X.Y.Z`) and merge the change to `main`.
+2. Create a new draft release on GitHub using the
+[Create Draft Release](https://github.com/MannLabs/alphashared/.github/workflows/create_release.yml) workflow.
+You need to specify the commit to release, and the release tag (e.g. `vX.Y.Z`).
+3. Test the release manually.
```

Comment:
Could you specify more? The create release workflow stores the releases somewhere (hidden folder?) and then you can test by downloading them and running them yourself?